### ISSUE TITLE
DM user URANUS

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#2151]
@SimHacker says DM should have a user named URANUS with password RINGS.

> 8) To play Zork, dial up the TIP after 8PM and connect to DM with "@L 70".
> 9) Log in as "URANUS" with password "RINGS".

https://news.ycombinator.com/item?id=29142947

**Comment by eswenson1** [comment]
What’s the point of this ticket? Are we planning on bringing up an ITS called DM and hosting it on the ‘net?

**Comment by larsbrinkhoff** [comment]
For now it's just information.  DM ITS is different from the rest in many ways, so I'm trying to keep notes about that.

**Comment by larsbrinkhoff** [comment]
Now I'm thinking having a URANUS user could actually serve a useful purpose.  The user can have a home directory, and it could have a \_MSGS\_ file in it to allow the user to play some games that check that.  Then we can document this so that people can log in as URANUS to play those games.

And yes, this might well only apply to a special "Dynamic Modeling" configuration.  E.g. DM had a password system (#2072) that we could enter URANUS into.  But I'm not sure it was still around in the Zork era, or if DM had adopted PANDA.

**Comment by larsbrinkhoff** [comment]
Hello @taa01776, @dlebling618, @jh95468,

Do you remember anything about a MIT-DMS user named URANUS, with password RINGS?

**Comment by taa01776** [comment]
Nope. UR was often prepended to initials for UROP students (I was initially URTAA), for what it's worth.
    -ta

Ceterum autem censeo Microsoftem esse delendam.
________________________________

**Comment by eswenson1** [comment]
You can log into ES from the ‘net and play the 616-point+100 point endgame Zork. Let me know if you want an account. 

We could easily host a system called DM, with a PWORD-based URANUS account with password RINGs on the ‘net if we wanted to. We could enable all the DM ITS functionality and run the DM daemons. I probably could get ITSDEM (pure library daemons), and COMBAT ZONE (compiler daemon), PCOMP, ASSEM, PDUMP, DBMAIN, etc running on that system. I have almost all of it running on ES. I think we should use PWORD rather than DM’s password system because the latter was not very secure and the former was transitioned later in the original DM’s lifetime (and PWORD was used on the other MIT ITS systems).

**Comment by eswenson1** [comment]
I just checked and I activated an account for you on ES back in October of 2020. So you already have an account (JFH). Send me email at eric at Swenson.org if you need me to reset your password (either you set it when you applied for an account back then) or I set it and emailed you the password if I proactively set up the account. Check your email for info on how to access ES.

**Comment by larsbrinkhoff** [comment]
I grepped all DM files (not too many unfortunately) for the string "uranus" and found nothing.  *However*, I checked the _TAPE backup records, and found some evidence.  Scant, but still.

```
 ACOUNT  APPLY   URANUS 1978-07-28 00:20:30
 COMMON  URANUS  SENDS  1979-07-13 00:48:37
 ACOUNT  HOLD    URANUS 1980-02-07 13:34:48
```

These are only file names stored in the _TAPE data, so I can't see the file contents.  But it certainly does look like someone applied for the URANUS user name 1978 and was active for a year or so.  Checking the other machines, I also see guest accounts on AI and ML in the same time period.  There is a mail file with the line "*Which of the many possible people(using the uranus account) are you?*" so it does seem as if this account had some illicit usage.

**Comment by taa01776** [comment]
"Illicit." The lab machines weren't exactly locked down at the best of times...
    -ta

Ceterum autem censeo Microsoftem esse delendam.
________________________________

**Comment by SimHacker** [comment]
I used it illicitly! ;)

(As I described in the Hacker News post linked above.)

https://news.ycombinator.com/item?id=29142947

At one point somebody from the lab sent me email saying they knew what I was doing and to stop. 

Then I actually started playing around with Macsyma on MC, and learning Lisp by running :TEACH;LISP, and KMP was very friendly and helpful in taking some time to tutor me personally, including showing me how "eval-in-other-lisp" let you send s-expressions to other people's lisp to do things like set their output base to roman numerals. 

https://news.ycombinator.com/item?id=20267415

Once I cursed at ANIMAL or DOCTOR and it got pissed off at me, so I apologized. (I think that email was archived somewhere but I can't find it now.)

Eventually I applied for my own account on DM, saying I wanted to learn to use MDL for algebraic applications (which not actually true, because I actually just wanted to play Zork, but that was what I was told to say), and it was approved.

The person who told me how to ask for an account on AI and DM was a guy named Bruce who ran a BBS running on a Northstar Horizon in Virginia. But I don't remember his last name or ITS user name. 

https://news.ycombinator.com/item?id=13519489

https://games.slashdot.org/comments.pl?sid=242967&cid=19681997
